src/views/documentListConst.ts

```diff
@@ -265,4 +266,31 @@ export const COLUMN_OPTIONS_MAP: ColumnOptionsMap = {
       classes: 'width-80px',
     },
   ],
+  'cloudfunctions': [
+    {
+      title: 'Name',
+      fieldNames: ['name'],
+      isSortable: true,
+      isSearchable: true,
+      classes: 'width-150px',
+    },
+    {
+      title: 'Description',
+      fieldNames: ['description'],
+      isSortable: true,
+      isSearchable: true,
+      classes: 'width-250px',
```

Comment:
Removed width from description so it would take as much room as possible
Added `ellipsis` for long descriptions

```suggestion
      classes: 'ellipsis',
```